tools/docker/cloud-node/Dockerfile

```diff
@@ -1,18 +1,61 @@
-ARG BUILDER_IMAGE=ghcr.io/build-trust/ockam/builder:latest
-ARG BASE_IMAGE=ghcr.io/build-trust/ockam/base:latest
+# Stage 1 - Download neccesary packages
+FROM debian:11.1-slim@sha256:312218c8dae688bae4e9d12926704fa9af6f7307a6edb4f66e479702a9af5a0c as dpkg
 
-# Stage 1
-FROM ${BUILDER_IMAGE}
+# We download all debs locally then extract them to the distroless container.
+WORKDIR /tmp
+RUN set -xe; \
+    apt-get update; \
+    apt-get download \
+        libc-bin; \
+    mkdir -p /dpkg/var/lib/dpkg/status.d/ && \
+    for deb in *.deb; do \
+        package_name=$(dpkg-deb -I ${deb} | awk '/^ Package: .*$/ {print $2}'); \
+        echo "Processing: ${package_name}"; \
+        dpkg --ctrl-tarfile $deb | tar -Oxf - ./control > /dpkg/var/lib/dpkg/status.d/${package_name}; \
+        dpkg --extract $deb /dpkg || exit 10; \
+    done
+
+# Remove unnecessary files extracted from deb packages like man pages and docs etc.
+RUN find /dpkg/ -type d -empty -delete && \
+    rm -r /dpkg/usr/share/doc/
+
+
+# Stage 2 - Build executable
+FROM ghcr.io/build-trust/ockam-builder@sha256:3ea4c222da220c69b62a60d42d83769286e07bf8e41ef7cfdbeba18b7f22a488 as executable
 COPY . /work
 RUN set -xe; \
     cd implementations/elixir; \
     ../../gradlew build; \
     cd ockam/ockam_cloud_node; \
     MIX_ENV=prod mix release;
 
-# Stage 2
-FROM ${BASE_IMAGE}
-COPY --from=0 /work/implementations/elixir/ockam/ockam_cloud_node/_build/prod/rel/ockam_cloud_node /opt/ockam_cloud_node
+
+# Stage 3 - Create distroless container and copy executables and packages in above steps
+FROM gcr.io/distroless/cc@sha256:3ca297cd5426268b5ad21e3fbe5c568411e0dec49dbae8e2967d33207bc99773
+
+COPY --from=executable /work/implementations/elixir/ockam/ockam_cloud_node/_build/prod/rel/ockam_cloud_node /opt/ockam_cloud_node
```

Comment:
@metaclips here's how you can test.

Replace `start` with `start_iex` [here](https://github.com/build-trust/ockam/blob/5f04709cffff82160a557ebdd64954e82a065588/tools/docker/cloud-node/Dockerfile#L61) then see if this works:
https://github.com/build-trust/ockam/blob/1e06dc70c401ccc216ede661c506cf08c1e03d05/examples/elixir/get_started/06-secure-channel-over-two-transport-hops-initiator.exs#L9-L11

Reply:
I get an ok response
<img width="1414" alt="Screenshot 2022-05-18 at 11 49 56 PM" src="https://user-images.githubusercontent.com/31141573/169168023-0325cb90-14f7-4f0f-ba89-9b788576ce38.png">

Reply:
That's great, this is likely because of the precompiled binaries is the priv folder.
This also means the FFI library file should also be release in the main release I mentioned here (added to that issue.)
https://github.com/build-trust/codebase/issues/216